data/roles/healers.json

```diff
@@ -0,0 +1,10 @@
+{
+    "title": "healers",
+    "name": "Healers",
+    "order": 2,
+    "color": "#61ce7e",
+    "icon": "/theme-assets/healer.png",
```

Comment:
Suggestion to change file name to "healers.png" for consistency with casters role.

Reply:
I agree, but probably change to `/theme-assets/icon/role/healer.png` (or perhaps `.svg` if we're lucky?)